worker-languagedetection-container/test-data/input-cld2/multi_field_multi_lang_en_fr_ger.yaml.testcase

```diff
@@ -0,0 +1,42 @@
+---
+tag: "multi_field_multi_lang_en_fr_ger.yaml"
```

Comment:
I'm confused by this change to a test case in the "input-cld2" directory, could you explain?

Reply:
This particular testcase was added for future comparison of detection against fasttext implementation, but as you asked to remove cld2 implementation completely, the testcases from input-cld2 are removed now.